server/handlers.go

```diff
@@ -12,6 +12,11 @@ import (
 	"github.com/gorilla/mux"
 )
 
+const (
+	// limit requests to only 32 mib to mitigate potential DoS attacks
+	maxRequestBodySize int64 = 1048576 * 32
```

Comment:
IMO `1024 * 1024 * 32` might be a clearer representation. Not sure `1048576` is common knowledge (I personally had to do the math to check it)

Reply:
[48c8cdc](https://github.com/Layr-Labs/eigenda-proxy/pull/227/commits/48c8cdc810a2c1dee50811e7f40413302d01422a)